codecov.yml

```diff
@@ -1,5 +1,11 @@
 coverage:
   status:
+    patch:
+      default:
+        target: 80%
+        threshold: 1%
+        base: auto 
+        if_ci_failed: error
```

Comment:
Solid. So just to be clear,

The `project` tag below asserts that the project stays at 80% coverage or above, and _this_ tag asserts that any PR changes are at least 80% covered by tests. Is that right?

Reply:
Correct.